platform/features/layout/src/FixedController.js

```diff
@@ -21,8 +21,22 @@
  *****************************************************************************/
 
 define(
-    ['./FixedProxy', './elements/ElementProxies', './FixedDragHandle'],
-    function (FixedProxy, ElementProxies, FixedDragHandle) {
+    [
+        'lodash',
+        './FixedProxy',
+        './elements/ElementProxies',
+        './FixedDragHandle',
+        '../../table/src/TelemetryCollection',
+        '../../../../src/api/objects/object-utils'
```

Comment:
These distant imports for TelemetryCollection and object-utils tell me we need to do some work on internal architecture. TelemetryCollection in particular seems like it does not belong in `table` any more, as it's equally useful for this view (and presumably others.) Where does it belong? That's what I don't think we have an answer for.

I don't think we should address it in the context of this PR; followup issue maybe?